app/controllers/admin/people_controller.rb

```diff
@@ -3,5 +3,32 @@ class PeopleController < AdminController
     def index
       @people = Person.all
     end
+
+    def batch_update
+      changes_made = false
```

Comment:
This has a bit of a smell to it. I'd recommend taking another look at how this is being implemented. If you're struggling with it fire me a message.